### PR TITLE
add doc page for “Map” global value

### PR DESCRIPTION
**Pull request by gdevillele**

### CODE REVIEW

File: lua-docs/content/reference/Map.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Returns the map's height, measured in cubes.
+
+### Depth (Integer, read-only)
+
+Returns the map's depth, measured in cubes.
+
+### Scale (Float, read-only)
+
+Returns the map's scale factor, measured in pixels per cube.
```
Comment (aduermael) [inline lua-docs/content/reference/Map.md]: You should only say `scale factor`. `scale == 1` doesn't mean `1 cube == 1 pixel`.

File: lua-docs/content/reference/Map.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Set (Function, read-only)
+
+Function to define the world map.
+
+Parameters:
+
+- map name (string)
+- scale factor (float)
```
Comment (aduermael) [inline lua-docs/content/reference/Map.md]: Is scale factor optional? If yes, we should indicate the default value. (8 I think)

File: lua-docs/content/reference/Map.md (hunk 1 above)
Comment (gdevillele) [reply lua-docs/content/reference/Map.md]: Right, I called this unit "pixel" in the implementation as I could not find a better name.
Changing this.

File: lua-docs/content/reference/Map.md (hunk 2 above)
Comment (gdevillele) [reply lua-docs/content/reference/Map.md]: As of today, it is not optional.

Comment (aduermael) [reply lua-docs/content/reference/Map.md]: 👍

**Pull request merged**